src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createDomNode = (type) => document.createElement(type);
+
+const addClassesToDomNode = (node, classes) => node.classList.add(...classes);
+
+const addTextToDomNode = (node, text) => (node.textContent = text);
+
+const appendNode = (parentNode, node) => parentNode.append(node);
+
+const createNotification = (type, parentNode, classes, text) => {
+  const node = createDomNode(type);
+
+  addClassesToDomNode(node, classes);
+
+  addTextToDomNode(node, text);
+
+  appendNode(parentNode, node);
+};
```

Comment:
The function `createNotification` is doing too much. It's creating a DOM node, adding classes, setting text, and appending the node to the parent. It might be better to split this function into smaller ones, each with a single responsibility. This will make your code more maintainable and easier to test.